app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @review = Review.new(params_review)
+    @review.user_id = current_user.id
+    @review.video_id = params[:video_id]
+    @review.rating = params[:Rating]
+
+
+    if @review.save
+      flash[:notice] = "Your review has been successfully saved."
+      redirect_to video_path(params[:video_id])
```

Comment:
If you do `@video = Video.find(params[:video_id])` as I mentioned above, you can use `redirect_to @video` :)